### DISCUSSION TITLE
Ip blocked

### DISCUSSION THREAD

**Discussion by Monub2008** [ppy/osu#15473] [Q&A]
I think my ip is blocked for some reason
Whenever i try to download any of the releases it says 'Download forbidden'. Can someone help me please?

**Comment by peppy** [comment]
That sounds like a github issue.

**Reply by Monub2008** [reply]
Then what do i do

**Reply by bdach** [reply]
we cannot help you based on a "download forbidden" message alone. we don't know whether it's github itself that returns that error, or if it's the browser, or if it's some antivirus software showing a false positive on your end.
